src/main/kotlin/org/jitsi/jicofo/xmpp/ConferenceIqHandler.kt

```diff
@@ -104,9 +104,11 @@ class ConferenceIqHandler(
     private fun processExtensions(query: ConferenceIq, response: ConferenceIq?, roomExists: Boolean): IQ? {
         val peerJid = query.from
         var identity: String? = null
+        val room = query.room
+        val isBreakoutRoom = room.toString().endsWith("@breakout.${XmppConfig.client.xmppDomain}")
```

Comment:
Maybe this should be configurable? Almost any component name is.

Reply:
Yes, almost any component name is currently configurable, but I'm wondering how many users are changing them at all? Even if it would be configurable it would be great to have a default, so there's less config properties to setup.

Reply:
The Docker setup does change them :-)

Reply:
Agree that this should be done, including adjustments in lib-jitsi-meet, jitsi-meet and prosody. Preferably I would do this in a separate PR introducing a customisation feature for the breakout rooms domain. Does that sound reasonable?

Reply:
Removed extra muc component definition for breakout rooms. They are using the main muc domain now. Breakout rooms are distinguished by a #breakout_uuid suffix.